go/service/config.go

```diff
@@ -161,6 +161,36 @@ func (h ConfigHandler) SetUserConfig(_ context.Context, arg keybase1.SetUserConf
 	return nil
 }
 
+func (h ConfigHandler) SetPath(_ context.Context, arg keybase1.SetPathArg) error {
+	svcPath := os.Getenv("PATH")
+	h.G().Log.Debug("SetPath: service path = %s", svcPath)
+	h.G().Log.Debug("SetPath: client path =  %s", arg.Path)
+
+	pathenv := strings.Split(svcPath, ":")
+	var clientAdditions []string
+NextDir:
+	for _, dir := range strings.Split(arg.Path, ":") {
+		for _, x := range pathenv {
```

Comment:
i know this is a rush job, but i'm dying inside!  can you use a map so it's not n^2? Thanks!!